App/Views/RenderView.swift

```diff
@@ -270,13 +270,23 @@ extension RenderView {
     
     /// Turns any links that look like tweets into an actual tweet embed.
     func embedTweets() {
-        webView.evaluateJavaScript("if (window.Awful) Awful.embedTweets()") { rawResult, error in
+        let renderGhostTweets = UserDefaults.standard.enableFrogAndGhost
+        webView.evaluateJavaScript("if (window.Awful) window.Awful.renderGhostTweets = \(renderGhostTweets); Awful.embedTweets()") { rawResult, error in
```

Comment:
I don't remember why the `if (window.Awful)` check is there (pretty sure I did it, and I probably had a reason, just don't remember :) but the `Awful.embedTweets()` bit here is outside of that check. Should put the two statements in `{` and `}`.

Reply:
Oh derp, nice catch. Sorted